test/e2e/specs/fakerp/metricsbridge.go

```diff
@@ -0,0 +1,133 @@
+package fakerp
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"net"
+	"net/http"
+	"regexp"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/ghodss/yaml"
+	"github.com/prometheus/client_golang/api"
+	"github.com/prometheus/client_golang/api/prometheus/v1"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift/openshift-azure/test/sanity"
+)
+
+type authorizingRoundTripper struct {
```

Comment:
consider moving this `authorizingRoundTripper` together with other roundtrippers we already use across the code into `pkg/util/rountrippers`

Reply:
done.